src/utils/handleResponse.js

```diff
@@ -1,18 +1,21 @@
 import newError from './errors';
 
 const handleResponse = (response) => {
+  if (!response.ok) {
```

Comment:
I think this statement should be moved inside the `if (response.headers.get('content-length') === '0' || response.status === 204) {`

Because now, when the server sends an error message with the 400 (so contentLength is not 0), it is not parsed (See line 16)